<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingRoom</class>
 <widget class="QWidget" name="SettingRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>671</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>检索</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_room_type">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_room_floor">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="widget_roomdetail" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(170, 255, 255)</string>
   </property>
   <widget class="QLabel" name="label_roomid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间ID号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_roomid">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomfloor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间楼层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomprice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间价格</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomtype">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomcount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间折扣</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_roomprice">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_roomcount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_floor">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_change">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_delete">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间名字</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_roomname">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomcount_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>房间电话</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_roomcall">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_newroom" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>261</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 127)</string>
   </property>
   <widget class="QLabel" name="label_roomfloor_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>房间楼层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomprice_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>房间价格</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomtype_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>房间类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomcount_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>房间折扣</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_roomprice">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_roomcount">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2_floor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2_type">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb2_add">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomfloor_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>房间名字</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_roomname">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_roomcount_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>房间电话</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2_roomcall">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
